src/openstudio_lib/CMakeLists.txt

```diff
@@ -716,7 +716,7 @@ add_library(${target_name}
   ${${target_name}_qrcs}
 )
 
-set(depends
+set(${target_name}_depends
```

Comment:
This looks like a decent change, why was it causing an issue?  Is there another variable named `depends`?

Reply:
Maybe this one `osapp\src\openstudio_app\CMakeLists.txt`?

Reply:
This is from https://github.com/openstudiocoalition/OpenStudioApplication/pull/182